Controllers/PizzaController.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using ContosoPizza.Models;
+using ContosoPizza.Services;
+
+namespace ContosoPizza.Controllers
+{
+    /// <summary>
+    /// Pizza Controller.
+    /// </summary>
+    /// <seealso cref="Microsoft.AspNetCore.Mvc.ControllerBase" />
+    [ApiController]
+    [Route("[controller]")]
+    public class PizzaController : ControllerBase
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PizzaController"/> class.
+        /// </summary>
+        public PizzaController()
+        {
+        }
+
+        /// <summary>
+        /// Gets all pizzas.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        public ActionResult<List<Pizza>> GetAll() =>
+            PizzaService.GetAll();
+
+        /// <summary>
+        /// Gets the specified pizza by identifier.
+        /// </summary>
+        /// <param name="id">The identifier.</param>
+        /// <returns></returns>
+        [HttpGet("{id}")]
+        public ActionResult<Pizza> Get(int id)
+        {
+            var pizza = PizzaService.Get(id);
+
+            if (pizza == null)
```

Comment:
Incorrect HTTP response code, if pizza not found, should return 404 Not Found, shouldn't return null